snap/README.md

```diff
@@ -0,0 +1,99 @@
+# Kata Containers snap image
+
+* [Kata Containers snap image](#kata-containers-snap-image)
+* [Initial setup](#initial-setup)
+* [Build snap image](#build-snap-image)
+* [Install snap (developer)](#install-snap-developer)
+* [Integration with docker](#integration-with-docker)
+* [Limitations](#limitations)
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu 18.04*
+
+```sh
+$ sudo apt-get install -y snapd snapcraft
+```
+
+## Build snap image
+
+Run next command at the root directory of the packaging repository.
+
+```sh
+$ make snap
+```
+
+## Install snap (developer)
+
+To install the resulting snap image, snap must be put in [classic mode][3] and the
+security confinement must be disabled (*--classic*). Also since the resulting snap
+has not been signed the verification of signature must be omitted (*--dangerous*).
+
+```sh
+$ sudo snap install --classic --dangerous kata-containers_[VERSION]_[ARCH].snap
+```
+
+Replace `VERSION` with the current version of Kata Containers and `ARCH` with
+the system architecture.
+
+## Configuring Kata Containers ##
+
+By default Kata Containers snap image is mounted at `/snap/kata-containers` as a
+read-only file system, therefore default configuration file can not be edited.
+Fortunately [kata-runtime][4] supports loading a configuration file from another
+path than the default.
+
+```sh
+$ sudo mkdir -p /etc/kata-containers
+$ sudo cp /snap/kata-containers/current/usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers/
+$ $EDITOR /etc/kata-containers/configuration.toml
+```
+
+## Integration with docker ##
```

Comment:
Should/can we point to the Ubuntu docker install directions here? - or add a note on how one follows but modifies those and then point at them? https://github.com/kata-containers/documentation/blob/master/install/docker/ubuntu-docker-install.md  Seems like duplication that we have tried to avoid for the other install guides?